lib/Command/Bot/Install.php

```diff
@@ -39,11 +39,6 @@ protected function configure(): void {
 				InputArgument::REQUIRED,
 				'The name under which the messages will be posted (min. 1 char, max. 64 chars)'
 			)
-			->addArgument(
```

Comment:
we could keep it, make URL optional and when URL is not proivded and the secret is starting with https:// while the `--secret` is provided, move the value over to URL. This would allow old commands to still work